ui/raidboss/data/04-sb/trial/tsukuyomi-ex.js

```diff
@@ -92,11 +92,12 @@
       },
     },
     {
-      regex: / 1A:\y{ObjectId}:Tsukuyomi gains the effect of Full Moon/,
-      regexDe: / 1A:\y{ObjectId}:Tsukuyomi gains the effect of Vollmond/,
-      regexFr: / 1A:\y{ObjectId}:Tsukuyomi gains the effect of Force De La Pleine Lune/,
-      regexJa: / 1A:\y{ObjectId}:ツクヨミ gains the effect of 満月流/,
-      regexCn: / 1A:\y{ObjectId}:月读 gains the effect of 满月流/,
+      regex: Regexes.gainsEffect({ effect: 'Full Moon', capture: false }),
```

Comment:
This needs `target: Tsukuyomi` (and a few others in this file)

Reply:
done